services/ble/Gatt.hpp

```diff
@@ -37,6 +37,20 @@ namespace services
         };
     };
 
+    namespace uuid
```

Comment:
@oguzcanoguz I believe these were previously (re)moved. What was the rationale back then, and is that still relevant now?

Reply:
Rationale was that we shall not put all those standard services in infralib. DIS is one and there can be surely others later.

Reply:
DIS is generic service and used widely unlike other standard services which are specific for the application, DIS can be part of ble services in infralib.

Reply:
@rjaegers @oguzcanoguz @magi-arun let me know if this issue can be resolved as won't fix, or if additional work needs to be done?

Reply:
As DIS is widely used BLE service, agreed to retain the dis uuid  as part of infralib. Closing this comment as won't fix.